tests/Integration/ParserTest.php

```diff
@@ -77,26 +78,17 @@ public function testParseFile()
     }
 
     /**
-     * Parsing certain PDFs may lead to following notices:
-     *
-     *      Notice: Trying to access array offset on value of type int
-     *
-     * and to an exception:
-     *
-     *      Missing catalog.
+     * Test that issue related pdf can now be parsed
      *
      * @see https://github.com/smalot/pdfparser/issues/267
      */
     public function testIssue267()
     {
-        $this->expectException(Exception::class);
-        $this->expectExceptionMessage('Missing catalog.');
-
         $filename = $this->rootDir.'/samples/bugs/Issue267_array_access_on_int.pdf';
 
         $document = $this->fixture->parseFile($filename);
 
-        // triggers the exception
-        $document->getPages();
+        self::assertEquals(Image::class, \get_class($document->getObjectById('128_0')));
+        self::assertStringContainsString('4 von 4', $document->getText());
```

Comment:
We usually use `$this->` instead of `self::`

```suggestion
        $this->assertEquals(Image::class, \get_class($document->getObjectById('128_0')));
        $this->assertStringContainsString('4 von 4', $document->getText());
```